crypto/zkp/primitives/dlogproofs/dlog_equality_blinded_transcript.go

```diff
@@ -106,12 +106,12 @@ func (prover *DLogEqualityBTranscriptProver) GetProofData(challenge *big.Int) *b
 	z := new(big.Int)
 	z.Mul(challenge, prover.secret)
 	z.Add(z, prover.r)
-	z.Mod(z, prover.DLog.GetOrderOfSubgroup())
+	z.Mod(z, prover.Group.Q)
 	return z
 }
 
-type DLogEqualityBTranscriptVerifier struct {
-	DLog       *dlog.ZpDLog
+type GroupEqualityBTranscriptVerifier struct {
```

Comment:
DLogEqualityBTranscriptVerifier should stay named as it is - it uses group, but it is for verification of dlog equality (group is only a place where dlog "happens").